milestoneupdater/config.json

```diff
@@ -32,23 +30,23 @@
   "dueDates": {
     "Nextcloud 27.1.10": "2024-05-23",
     "Nextcloud 27.1.11": "2024-06-20",
-    "Nextcloud 28.0.6": "2024-05-23",
     "Nextcloud 28.0.7": "2024-06-20",
-    "Nextcloud 29.0.1": "2024-05-23",
     "Nextcloud 29.0.2": "2024-06-20"
```

Comment:
that will be true for 29.0.3, 29.0.2 gets the out-of-band release next week

Reply:
Updated